core/dbt/tests/fixtures/project.py

```diff
@@ -356,8 +356,8 @@ def project_files(project_root, models, macros, snapshots, properties, seeds, te
 
 # We have a separate logs dir for every test
 @pytest.fixture(scope="class")
-def logs_dir(request, prefix):
-    return os.path.join(request.config.rootdir, "logs", prefix)
+def logs_dir(project_root, prefix):
+    return os.path.join(project_root, "logs", prefix)
```

Comment:
Ah! I didn't realize this was the desired behaviour but I can see the utility. I changed it because this `logs_dir` is taking precedence over the default, and we're [expecting the default log location for in a functional test](https://github.com/dbt-labs/dbt-core/pull/6908/files#diff-5c2acd7b48060d11487f9bbb724169956b01dc7daf12d4e62a29dd03ae673ab4R337). I'll do a bit more digging here to understand what main does better and find an alternative.

Reply:
Okay - I see what was going on. The test was relying on a default location of log path that was incorrect on the feature branch. When we fixed the default location of log path and properly read it from the dbt_project.yml, this test started to break again. The fix is just to read from `logs_dir` directly in the test, since that's where the tests are written. 

Thanks for pointing this out!